src/deepsparse/image_classification/schemas.py

```diff
@@ -32,12 +32,13 @@
 
 class ImageClassificationInput(BaseModel, Splittable):
     """
-    Input model for image classification
+    Input model for image classification, the images can be a str file path,
+    a List of file paths, List of numpy arrays or a single numpy array
     """
 
-    images: Union[str, List[str], List[Any]] = Field(
+    images: Union[str, List[str], List[Any], Any] = Field(
         description="List of Images to process"
-    )  # List[Any] to accept List[numpy.ndarray]
+    )  # List[Any] to accept List[numpy.ndarray], Any to accept numpy.ndarray
```

Comment:
Can we add some unit tests for all of these?

```
def test_accepts_input():
    ImageClassificationInput("asdf")
    ImageClassificationInput(["asdf", "qwer"])
    ImageClassificationInput(np.zeros((1, 2, 3))
    ImageClassificationInput([np.zeros((1, 2, 3), np.zeros((1,2,3))])
```

Reply:
Thanks! 
Instead of adding similar tests for all three schemas I'd like to refactor common code from CV schemas into a parent class and test just that; Will be adding in those changes and tests

Reply:
Added tests!